ext/web/10_filereader.js

```diff
@@ -421,39 +469,30 @@
     wrappedHandler.handler = handler;
     return wrappedHandler;
   }
-  // TODO(benjamingr) reuse when we can reuse code between web crates
-  function defineEventHandler(emitter, name) {
-    // HTML specification section 8.1.5.1
-    ObjectDefineProperty(emitter, `on${name}`, {
-      get() {
-        const maybeMap = this[handlerSymbol];
-        if (!maybeMap) return null;
-
-        return MapPrototypeGet(maybeMap, name)?.handler ?? null;
-      },
-      set(value) {
```

Comment:
same as getter,  ObjectDefineProperty set would fail at https://github.com/web-platform-tests/wpt/blob/51d8c24e16deaf156a7eab496cf06721e55c4ebf/resources/idlharness.js#L2994